playlist_length/main.py

```diff
@@ -76,34 +78,21 @@ def calculate_length(BASE_PATH, no_subdir, media_type):
 
     all_files = get_all_files(BASE_PATH, no_subdir)
 
-    with ProcessPoolExecutor() as executor:
-        sys.stdout.write('\n')
-        video_files = []
-        tasks = [executor.submit(is_media_file, file_path) for file_path in all_files]
-
-        for task in tqdm(
-            as_completed(tasks), total=len(tasks), desc='Filtering {}'.format(media_type)
-        ):
-            path = task.result()
-            if path is not None:
-                video_files.append(path)
-
-    if not video_files:
-        return bold(red('Seems like there are no {} files. ¯\_(ツ)_/¯'.format(media_type)))
-
     with ProcessPoolExecutor() as executor:
         sys.stdout.write('\n')
         result = list(
             tqdm(
-                executor.map(duration, video_files),
-                total=len(video_files),
-                desc='Calculating time',
+                executor.map(duration, all_files),
+                total=len(all_files),
+                desc='Processing files',
```

Comment:
We are calculating the time right?

Reply:
it will look odd if there are no media files while the description of bar being 'Calculating media files'